blockchain/tx_pool.go

```diff
@@ -54,6 +55,8 @@ var (
 	statsReportInterval = 8 * time.Second // Time interval to report transaction pool stats
 
 	txPoolIsFullErr = fmt.Errorf("txpool is full")
+
+	errNotAllowedTx = errors.New("not allowed tx")
```

Comment:
```suggestion
	errNotAllowedTx = errors.New("locally anchoring chaindata tx is not allowed in this node")
```

Reply:
@ehnuje I intended not to write the transaction type on this error for using commonly.
Instead of that, I added the tx type when the error is returned.